service/index.ts

```diff
@@ -135,14 +135,14 @@ const api = {
     method: 'POST',
     url: paths.orderFormShipping,
     headers: profileCustomHeaders('application/json'),
-    data: ({fields}) => merge({expectedOrderFormSections: ['items']}, fields)
+    data: (data) => merge({expectedOrderFormSections: ['items']}, data)
   }),
 
   '/mutation/updateOrderFormIgnoreProfile': handleEndpoint({
     method: 'PATCH',
     url: paths.orderFormIgnoreProfile,
     headers: profileCustomHeaders('application/json'),
-    data: ({fields}) => merge({expectedOrderFormSections: ['items']}, fields)
+    data: ({ignoreProfileData}) => merge({expectedOrderFormSections: ['items']}, ignoreProfileData)
```

Comment:
I think this merge won't work as you want it to, since `ignoreProfileData` isn't an object, so its information will be lost here. This probably needs to change to be something like:

`data: ({ignoreProfileData}) => ({expectedOrderFormSections: ['items'], ignoreProfileData})`